skin-resque-frontend/src/components/Pagination/Pagination.js

```diff
@@ -0,0 +1,31 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Heading from '../Heading/Heading';
+import './Pagination.scss';
+
+export default function Pagination({ color, size }) {
+
+	const pages = (new Array(size)).fill(0);
+
+	const pagesList = pages.map((_, index) => {
```

Comment:
You can merge this and the one above into one constant. I think it look better but I'm leaving it up to you